src/providers/realtime-provider.js

```diff
@@ -205,7 +234,20 @@ export default class RealtimeEventsProvider {
         }
     }
 
+    getSubscriptionDetailsById(id) {
+        return Array.from(this.subscriptions.values()).find(
```

Comment:
This seems like something we should use a map for. I think I wrote this code, so blame me.

Reply:
It's @akhenry 's fault. I'll fix it though... :)